dna/clutter/clutter.js

```diff
@@ -58,6 +58,20 @@ function postMod(params) {
     return update("post",post,hash);
 }
 
+function getPost(params) {
```

Comment:
Before merging this I'd love for this function to have a test added to [tests/posts.json](https://github.com/Holochain/clutter/blob/master/test/posts.json)  Would you like to add that, or should I?